sensor.cpp

```diff
@@ -157,7 +157,7 @@ static const Sensor::ButtonMap ikeaRemoteMap[] = {
 //    mode                          ep    cluster cmd   param button                                       name
     // big button
     { Sensor::ModeColorTemperature, 0x01, 0x0006, 0x02, 0,    S_BUTTON_1 + S_BUTTON_ACTION_SHORT_RELEASED, "Toggle" },
-    { Sensor::ModeColorTemperature, 0x01, 0x0005, 0x07, 2,    S_BUTTON_1 + S_BUTTON_ACTION_HOLD,           "Setup 10s" },
+    // { Sensor::ModeColorTemperature, 0x01, 0x0005, 0x07, 2,    S_BUTTON_1 + S_BUTTON_ACTION_HOLD,           "Setup 10s" },
```

Comment:
I agree with the reasoning that the HOLD action isn't a good fit here. However in current Phoscon App versions the pairing wizard for this remote requires this button event to finish the procedure. The next gen. Phoscon App wizard is happy to accept any button event, but it's not released yet.

I would suggest we use either another existing action like INITIAL_PRESS or add a new action to capture the event?

Reply:
I was afraid some-one or something would depend on this button event.  Best leave it for now, and pick it up once new Phoscon release is out.

Imho, sending INITIAL_PRESS is just as bad as sending HOLD: client applications expect a SHORT_RELEASED or HOLD after INITIAL PRESS, just as they expect a LONG_RELEASED after HOLD.  As discussed in the issue, we could map it to LONG_RELEASED, but that's still kinda weird without an INITIAL_PRESS or HOLD.  Then again, client applications polling the Hue bridge might miss these and only see the LONG_RELEASED.